packages/site-components/src/Sidebar/styles.css.ts

```diff
@@ -4,6 +4,7 @@ import { sidebar } from '@jpmorganchase/mosaic-theme';
 export default {
   root: style([
     {
+      width: '400px',
```

Comment:
Does this impact the responsive design in any way?  When I looked the sidebar goes full width on smaller devices so I guess not?

Reply:
yes , widths are driven via theme, so adding this might have some impact.